user-carbon/src/utils/carbon_token/ICarbonToken.ts

```diff
@@ -0,0 +1,8 @@
+import { Address, BigDecimal, BigInt } from "@graphprotocol/graph-ts";
+
+export interface ICarbonToken {
+    getDecimals(): number
+    getAddress(): Address
+    getToken(): string
+    returnAddressBalance(address: Address): BigDecimal
```

Comment:
Maybe we could reuse already existing `IToken` interface (now when refactoring it's merged and just include `returnAddressBalance` as a new method there? Tokens are put within `libs` folder so we can share it within different subgraphs